init_configure_commands_tut.md

```diff
@@ -0,0 +1,156 @@
+    - The `.config` directory consists of the metadata files tied to your project. The `project-config.json` file has the following format
+        ```
+        {
+            "projectName": "test890",
+            "projectPath": "/Users/kaustavg/test890",
+            "defaultEditor": "sublime",
+            "frontendHandler": {
+                "javascript": "/Users/xxxxx/aws-amplify-cli/amplify-cli/packages/amplify-cli/node_modules/amplify-frontend-javascript"
+            },
+            "javascript": {
+                "framework": "none",
+                "config": {
+                    "SourceDir": "src",
+                    "DistributionDir": "dist",
+                    "BuildCommand": "npm run-script build",
+                    "StartCommand": "npm run-script start"
+                }
+            },
+            "providers": {
+                "awscloudformation": "/Users/xxxxx/aws-amplify-cli/amplify-cli/packages/amplify-cli/node_modules/amplify-provider-awscloudformation"
+            }
+        }
+        ```
+        Note: The 'project-config.json' file has your local system specific paths and shouldn't be checked in, into yoru source control if sharing the project with anothe team member working on another system.
+        
+        The `.config` directory also has `aws-info.json` file which lets the CLI know which AWS profile/accesskey-secret key pair to use when adding AWS resources to your project. If you're using an AWS profile to initialize your project, the format should be the following
+        
+        ```
+        {
+            "useProfile": true,
+            "profileName": "amplifyprofile"
+        }
+        ```
+    
+    - The `.amplifyrc` file consists of deployment-related information which is not currently used but would be used in the future for CI/CD workflows. It should have the following format.
+         ```
+            {
+                "providers": {
+                    "awscloudformation": {
+                        "AuthRoleName": "test890-xxxxxxx-authRole",
+                        "UnauthRoleArn": "arn:aws:iam::132393967379:role/test890-xxxxx-unauthRole",
+                        "AuthRoleArn": "arn:aws:iam::132393967379:role/test890-xxxxxxxx-authRole",
+                        "Region": "us-east-1",
+                        "DeploymentBucketName": "test890-xxxxxxx-deployment",
+                        "UnauthRoleName": "test890-xxxxxx-unauthRole",
+                        "StackName": "test890-xxxxxxxxx",
+                        "StackId": "arn:aws:cloudformation:us-east-1:132393967379:stack/test890-20181006145850/fe8b16d0-xxxxxxxxxxx"
+                    }
+                }
+            }
+        ```
+        
+## amplify configure project
+```bash
+$ amplify configure project
+```
+The `amplify configure project` command should be used in the following cases:
```

Comment:
I would change to: "The amplify configure project is an advanced command and not commonly used for getting started or individual development projects. The command should be used in the following cases:"